amqp-protocol-binding.md

```diff
@@ -186,8 +174,29 @@ Examples:
 ##### 3.1.3.2 AMQP Application Property Values
 
 The value for each AMQP application property is constructed from the respective
-attribute's AMQP type representation, compliant with the [AMQP event
-format][amqp-format] specification.
+attribute's AMQP type representation.
+
+The CloudEvents type system MUST be mapped to AMQP types as follows, with
+exceptions noted below.
+
+| CloudEvents   | AMQP                        |
+| ------------- | --------------------------- |
+| Boolean       | [boolean][amqp-boolean]     |
+| Integer       | [long][amqp-long]           |
+| String        | [string][amqp-string]       |
+| Binary        | [binary][amqp-binary]       |
+| URI           | [string][amqp-string]       |
+| URI-reference | [string][amqp-string]       |
+| Timestamp     | [timestamp][amqp-timestamp] |
+
+All attribute values in an AMQP binary message MAY be represented using the
+native AMQP types above, or MAY be in canonical string form. An implementation
+MUST be able to interpret both forms on an incoming AMQP message. An
+implementation SHOULD use the native AMQP form on outgoing AMQP messages when it
+is efficient to do so, but MAY forward values as canonical strings. A robust
+implementation MAY further relax the requirements for incoming messages (for
+example accepting numeric types other than AMQP long) but MUST use only the
```

Comment:
is the MUST at the end in conflict with the MAY in the first line?  "MUST use only AMQP types" vs "MAY use AMQP types or MAY use strings"